### ISSUE TITLE
Links are not clearly identifiable (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#134]
122 occurrences of this issue that should be one fix in the CSS. All links she be underlined by default. 

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [ ] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@tubesoft : At the Newton meeting today, Bill said you would be working on these accessibility issues. I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

Could you have a look at this issue and confirm it's something you can fix? Once you confirm, we can figure out a testing workflow.

**Comment by tubesoft** [comment]
I am adding `text-decoration:underline;` for `<a>` tags in `global.css`.
<img width="365" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/27016e17-df9e-4571-8c7a-61aeb1cfabdf">

**Comment by tubesoft** [comment]
@mdalmau 

Before:
<img width="664" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/c0bb09ff-de40-4150-84fc-ba7ea9437538">

After:
<img width="668" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/c573a429-d118-459c-ba03-bad995190784">

However, the browser plugin told me that some other parts were not covered by this CSS code. I will investigate it further.

**Comment by mdalmau** [comment]
@tubesoft : thank you! I just wanted to make sure that Bill & Co. could have a sneak peak. We have the page number links in the rendered manuscripts that may not be covered ... I am not sure. Thanks for looking into this.

**Comment by mdalmau** [comment]
Since this issue is now part of https://github.com/IUBLibTech/newton_chymistry/pull/146, I will mark this issue as "done" so I can better see the remaining accessibility issues!
